wg-batch/annual-report-2024.md

```diff
@@ -2,27 +2,132 @@
+
+Topology aware scheduling facilitates scheduling of workloads that take in account data center topology. Workloads benefit from using interconnects that are physically close together.
+
+MultiKueue provides a way of dispatching batch workloads to worker clusters. Kueue provides multicluster dispatching for popular batch workloads such as Ray, Job, Kubeflow and JobSet. This feature went beta in 0.9.
+
+Kueue Dashboards has been a popular ask for Kueue. Users would like to have a visualization representation of queueing and we are happy to announce that a dashboard has been created for Kueue. This went into kueue in late 2024 and a big focus of 2025 will be to harden this for production.
+
+KueueCtrl provides a cli for creating kueue objects. The plugin is hosted in krew and is easily installed as a kueue plugin.
+
+Deployment/StatefulSet integration provides an avenue for the usage of Kueue for serving workloads. Serving leads to a need for sharing/preemption of model servers that may leverage accelerators. Kueue provides an integration with popular methods of deploying services (Deployment/StatefulSet).
+
+##### JobSet
+
+Jobset has had 4 release in 2024.
+
+- [Release 0.4](https://github.com/kubernetes-sigs/jobset/releases/tag/v0.4.0)
+
+- [Release 0.5](https://github.com/kubernetes-sigs/jobset/releases/tag/v0.5.0)
+
+- [Release 0.6](https://github.com/kubernetes-sigs/jobset/releases/tag/v0.6.0)
+
+- [Release 0.7](https://github.com/kubernetes-sigs/jobset/releases/tag/v0.7.0)
+
+A major achievement of JobSet has been the adoption of JobSet as a component for Kubeflow Training Operator V2.
+There has been a collaborative effort with the Kubeflow community and the batch community to implement the features needed for this integration.
+
+[Metaflow](https://github.com/Netflix/metaflow/pull/1804) has adopted the use of JobSet for distributed ML training.
+
+##### KJob
+
+[KJob](https://github.com/kubernetes-sigs/kjob?tab=readme-ov-file#kjob) has been started to provide a CLI friendly way for users to submit batch jobs.
+The HPC/ML community tend to prefer CLI over YAML so the focus was to provide a templated solution for submitting batch jobs.
+Another focus of this project is to provide a smooth transition for Slurm users.
+
+#### KEPs
+
+WG-Batch provided a series of kubernetes enhancements that improved the experience of batch workloads on Kubernetes. In 2024, this group proposed/implemented the following KEPs.
+
+- [Job Managed By](https://github.com/kubernetes/enhancements/issues/4368)
+  - Promoted to beta in 2024
+
+- [Job Success Policy](https://github.com/kubernetes/enhancements/issues/3998)
+  - Promoted to beta.
+
+- [Elastic Index Jobs](https://github.com/kubernetes/enhancements/issues/3715)
+  - Promoted to stable.
+
+- [Pod Failure Policy](https://github.com/kubernetes/enhancements/issues/3329)
+  - Promoted to stable.
+
+- [Pod Index Label](https://github.com/kubernetes/enhancements/issues/4017)
+  - Promoted to stable.
+
+### Talks
```

Comment:
This is great resource!

nit: the format could be more consistent:
- decide on "and" or &. 
- 2024 seems repetative in the context
- "KubeCon EU, March, Paris" vs "Kubecon EU 2024"

Reply:
good call. Updated.